CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 # Change Log
+## 1.19.0 - 2019-11-xx
+### Added
+* Add docker status detection
+* Support Azure Event Grid
```

Comment:
Be more clear about this feature. Please sync with Jun about the wording

Reply:
OK, just sync with Jun, we will change it to "Support adding Azure Event Grid Module"